vlasovsolver/cpu_moments.cpp

```diff
@@ -210,6 +217,7 @@ void calculateMoments_R(
                                       array);
           } // for-loop over velocity blocks
 
+          if (array[0] == 0) {std::cout << "In calculateMoments_R, array[0] = " << array[0] << " for cellid " << cell->parameters[CellParams::CELLID] << ". This cell contains " << blockContainer.size() << " blocks" << std::endl;}
```

Comment:
delete this line (we don't want these printouts as it's viable to have zero populations in some parts of the domain for multipop)